content/applications/inventory_and_mrp/manufacturing/subcontracting/subcontracting_lead_times/basic_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,101 @@
+the *Expected Arrival* date displayed on purchase orders (POs) for the product will more accurately
+reflect the amount of time required for both manufacturing and delivery.
+
+Product delivery lead time
+--------------------------
+
+To set a *delivery lead time* for a product's subcontractor, navigate to :menuselection:`Inventory
+app --> Products --> Products`, and select a subcontracted product.
+
+Then, select the :guilabel:`Purchase` tab on the product's page. If the subcontractor has not
+already been added as a vendor, do so now by clicking :guilabel:`Add a line`, and selecting the
+subcontractor in the :guilabel:`Vendor` field.
+
+Once the subcontractor has been added, enter the number of days it takes them to manufacture and
+deliver the product, in the :guilabel:`Delivery Lead Time` field.
+
+.. image:: basic_subcontracting_lead_times/delivery-lead-time.png
+   :align: center
+   :alt: The Delivery Lead Time field for a subcontractor, on the Purchase tab of a product page.
+
+.. note::
+   Multiple subcontractors can be added to the :guilabel:`Purchase` tab on a product's page, and a
+   different :guilabel:`Delivery Lead Time` can be set for each.
+
+Lead time workflow
+==================
+
+After setting a *delivery lead time* for a product's vendor, create an |RfQ| by navigating to
+:menuselection:`Purchase app --> Orders --> Purchase Orders`, and clicking :guilabel:`New`.
+
+Specify the subcontractor in :guilabel:`Vendor` field. Then, add the product in the
+:guilabel:`Products` tab by clicking :guilabel:`Add a product`, selecting the product in the
+:guilabel:`Product` field, and adding a quantity in the :guilabel:`Quantity` field.
+
+Once a product has been added, the :guilabel:`Expected Arrival` field on the |RfQ| auto-populates
+with a date that reflects the *delivery lead time* specified on the product's page.
+
+If the date needs to be adjusted, click on the :guilabel:`Expected Arrival` field to open a calendar
+pop-up, and select the desired date. Make sure not to choose a date sooner than the one that was
+auto-populated, unless the subcontractor has confirmed that they are able to deliver the product by
+that date.
+
+Finally, click :guilabel:`Confirm` on the |RfQ| to turn it into a |PO|. At this point, the
+subcontractor should begin manufacturing the subcontracted product, before delivering it by the
+*Expected Arrival* date.
+
+.. example::
+   Bike retailer `Mike's Bikes` works with a subcontractor — `Bike Friends` — to produce units of
+   their `Tricycle` product.
+
+   On average, `Bike Friends` requires three days to manufacture a `Tricycle`, plus an additional
+   two days to deliver it to `Mike's Bikes`.
+
+   As a result, `Mike's Bikes` sets a *delivery lead time* of five days for `Tricycles` manufactured
+   by `Bike Friends`: three days for manufacturing, plus two days for delivery.
+
+   On May 3rd, `Mike's Bikes` confirms a |PO| to purchase one `Tricycle` from `Bike Friends`.
```

Comment:
I'm not an expert on technical reviews, but maybe these terms should be entered as italics and not input text, which the `` indicates.

Reply:
Maybe using gui label because we see some of these labels on the screen shot.